src/Sylius/Bundle/WebBundle/Menu/FrontendMenuBuilder.php

```diff
@@ -260,8 +260,8 @@ public function createTaxonsMenu()
         foreach ($taxons as $taxon) {
             $child = $menu->addChild($taxon->getName(), $childOptions);
 
-            if ($taxon->hasPath()) {
-                $child->setLabelAttribute('data-image', $taxon->getPath());
+            if ($taxon->hasImages()) {
+                $child->setLabelAttribute('data-image', $taxon->getImages()->first()->getPath());
```

Comment:
i'm :+1: with `$taxon->(has|get)ImageWithCode('thumbnail')`

And maybe `getFirstImage()` which is pretty much the same as `getDefaultImage()` WDYT?